mongodb/src/main/scala/akka/stream/alpakka/mongodb/MongoConnectionDetails.scala

```diff
@@ -0,0 +1,9 @@
+package akka.stream.alpakka.mongodb
+
+import org.mongodb.scala.{MongoClient, MongoDatabase}
+
+private[mongodb] case class MongoConnectionDetails(dbName: String,host: String, port:Int) {
+  lazy val client: MongoClient = MongoClient(s"mongodb://$host:$port")
+
+  lazy val db: MongoDatabase = client.getDatabase(dbName)
+}
```

Comment:
This is only used in the tests so can be moved there, maybe even inlined in the spec?